mp/src/game/shared/momentum/weapon/weapon_blank.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "weapon_base.h"
+
+#if defined(CLIENT_DLL)
+#define CBlank C_Blank
+#endif
+
+// ----------------------------------------------------------------------------- //
+// CBlank class definition.
+// ----------------------------------------------------------------------------- //
+
+class CBlank : public CWeaponBase
+{
+  public:
+    DECLARE_CLASS(CBlank, CWeaponBase);
+    DECLARE_NETWORKCLASS();
+
+    CBlank();
+
+    // We say yes to this so the weapon system lets us switch to it.
+    bool HasPrimaryAmmo() OVERRIDE { return true; }
```

Comment:
Use lower case `override`. This macro was for Linux support I believe but now it is not needed.